traffic_ops/app/lib/API/Configs/ApacheTrafficServer.pm

```diff
@@ -2617,13 +2617,8 @@ sub parent_dot_config { #fix qstring - should be ignore for quika
 				}
 				$text .= "$parents round_robin=$mso_algorithm qstring=$parent_qstring go_direct=false parent_is_proxy=false";
 
-				if ( $ats_major_version >= 6 && $parent_retry ne "" ) {
-					if ( $unavailable_server_retry_responses ne "" && $unavailable_server_retry_responses =~ /^"(?:\d{3},)+\d{3}"\s*$/) {
```

Comment:
You removed the validation this does on the param - it used to check for a valid 3-digit response code.

Reply:
@ocket8888 Would you know what happens if not validated? Traffic Server crash? If so, should have been documented. 

I removed the validation because there was now defaults for all these params. 

What should be the behavior if I add the validation? Add the "503" default and not fail?

Reply:
Well the old behavior was to fall back on just `parent_retry=$parent_retry`, so you could do that. But you could also just fall back on the default. In fact, that's probably more appropriate since it used to fall back on `parent_retry=$parent_retry` if it failed validation _or_ if it wasn't defined. So the most consistent thing to do would be adding the 503 and not fail - but you may want to log a warning.

As far as what would happen to ATS, I believe that in general parsing failures would cause either the field or the line to fail, depending on the ATS version and the file in question. Meaning it wouldn't crash, but the configuration won't be applied and it will log an error.